src/fragments/lib/auth/flutter/getting_started/70_configureBackend.mdx

```diff
@@ -0,0 +1,155 @@
+<BlockSwitcher>
+<Block name="Amplify CLI (Create)">
+
+> Prerequisites: [Install and configure](/lib/project-setup/prereq/q/platform/flutter/#install-and-configure-the-amplify-cli) the Amplify CLI in addition to the Amplify libraries and [necessary dependencies](/lib/project-setup/create-application/q/platform/flutter/#2-install-amplify-libraries).
+
+To start provisioning auth resources in the backend, go to your project directory and **execute the command**:
+
+```bash
+amplify add auth
+```
+
+Enter the following when prompted:
+```console
+? Do you want to use the default authentication and security configuration?
+    `Default configuration`
+? How do you want users to be able to sign in?
+    `Username`
+? Do you want to configure advanced settings?
+    `No, I am done.`
+```
+
+> If you have previously enabled an Amplify category that uses Auth behind the scenes (e.g. API category), run the `amplify update auth` command to edit your configuration. 
+
+To push your changes to the cloud, **execute the command**:
+
+```bash
+amplify push
+```
+
+import ios3 from "/src/fragments/lib/auth/ios/getting_started/12_amplifyConfig.mdx";
+
+<Fragments fragments={{ios: ios3}} />
+
+import android4 from "/src/fragments/lib/auth/android/getting_started/12_amplifyConfig.mdx";
+
+<Fragments fragments={{android: android4}} />
+
+import flutter5 from "/src/fragments/lib/auth/flutter/getting_started/12_amplifyConfig.mdx";
+
+<Fragments fragments={{flutter: flutter5}} />
+
+</Block>
+
+<Block name="Amplify CLI (Import)">
+
+> Prerequisites: [Install and configure](/lib/project-setup/prereq/q/platform/flutter/#install-and-configure-the-amplify-cli) the Amplify CLI in addition to the Amplify libraries and [necessary dependencies](/lib/project-setup/create-application/q/platform/flutter/#2-install-amplify-libraries).
+
+To import existing Amazon Cognito resources into your Amplify project, **execute the command**:
+
+```bash
+amplify import auth
+```
+```console
+? What type of auth resource do you want to import?
+   Cognito User Pool and Identity Pool
+   Cognito User Pool only
+```
+
+Once you've selected an option, you'll be able to search for and import an existing Cognito User Pool and Identity Pool (or User Pool only) within your AWS account.  The `amplify import auth` command will also do the following:
+- Automatically populate your Amplify Library configuration files (`amplifyconfiguration.dart` and `amplifyconfiguration.json`) with your chosen Amazon Cognito resource information
+- Provide your designated existing Cognito resource as the authentication & authorization mechanism for all auth-dependent categories (API, Storage and more)
+- Enable Lambda functions to access the chosen Cognito resource if you permit it
+
+> If you have previously enabled an Amplify category that uses Auth behind the scenes (e.g. API category), run `amplify update auth` command to edit your configuration. 
```

Comment:
I would remove this - I really don't understand it in the context of importing.

Reply:
Updated in most recent commit.  I went ahead and removed it as an unordered list/bullet point since it doesn't makes sense with just 1.